src/test/java/ladder/domain/game/ResourceGenerator.java

```diff
@@ -0,0 +1,37 @@
+package ladder.domain.game;
+
+import java.util.ArrayList;
+import java.util.List;
+import ladder.domain.resource.ladder.Ladder;
+import ladder.domain.resource.ladder.LadderGenerator;
+import ladder.domain.resource.line.RandomLineGenerator;
+import ladder.domain.resource.prize.Prize;
+import ladder.domain.resource.prize.Prizes;
+import ladder.domain.resource.user.User;
+import ladder.domain.resource.user.Users;
+
+public class ResourceGenerator {
+
+    public static Users generateUsersWithCount(int count) {
```

Comment:
```suggestion
    public static Users generateUsers(int count) {
```
꼭 적지 않아도 되는 내용은 과감하게 없애도 메서드가 어떻게 행동할지 이해할 수 있을 것 같아요!